README.md

```diff
@@ -1,2 +1,12 @@
-# hw01
-Repository for homework 1
+#Carman Fowler Biography
+##Personal
+###I am from Camden, South Carolina.
+###![Camden Sign](http://maineanencyclopedia.com/wp-content/uploads/camden01.jpg)
+###My hobbies include: 
+###1. *Creative writing*
+###2. Going on *ghost tours*
+###3. Playing *Dungeons and Dragons*. 
+###This is my first year in Chicago and I haven't seen a lot of the city but my favorite place to go is the Art Institute. 
+##Research
+###I am in the **MAPSS** program and I work with **Dr. Ayelet Fishbach** in the **Booth School**. My research interests involve moral decision making, cognitive dissonance, and effects of culture and identity. 
+#[Carman W Fowler](https://www.linkedin.com/in/carmanwfowler/)
```

Comment:
This link appeared properly both in the preview and on GitHub.

Reply:
Great job on the link. For a while, mine wasn't rendering properly as a hyper link. finally I realized the space between the ] and ( was causing the problem. clearly you figured that out.